Source/WebKit/NetworkProcess/storage/CacheStorageManager.cpp

```diff
@@ -223,15 +223,17 @@ void CacheStorageManager::makeDirty()
     m_updateCounter = nextUpdateNumber();
 }
 
-String CacheStorageManager::saltFilePath() const
+FileSystem::Salt CacheStorageManager::readOrMakeSalt() const
```

Comment:
This does not have to be a class method. Let's make it:
`static Filesystem::Salt readOrMakeSalt(const String& saltDirectory)` and `readOrMakeSalt(m_path)` below

Reply:
Thanks, addressed.